ppfleetx/models/vision_model/vit/vit.py

```diff
@@ -205,6 +288,76 @@ def forward(self, x):
         x = self.head(x)
         return x
 
+    def state_dict(self,
+                   destination=None,
+                   include_sublayers=True,
+                   structured_name_prefix="",
+                   use_hook=True):
+        state_dict = super().state_dict(destination, include_sublayers,
```

Comment:
The `qkv.weight` in ViTAttention is of the shape `[embed_dim, embed_dim]`, but `attn.qkv_weight` in FusedAttention is of the shape `[3, num_head, head_dim, embed_dim]`. There should have a transpose between `qkv.weight` and `attn.qkv_weight`.